source/includes/steps-install-shell-linux-deb.yaml

```diff
@@ -1,21 +1,126 @@
+title: Import the public key used by the package management system.
+stepnum: 1
+level: 4
+ref: import-key
+content: |
+    From a terminal, issue the following command to import the
+    MongoDB public GPG Key from `<https://www.mongodb.org/static/pgp/server-{+pgp-version+}.asc>`_:
+
+    .. code-block:: sh
+
+       wget -qO - https://www.mongodb.org/static/pgp/server-{+pgp-version+}.asc | sudo apt-key add -
+
+    The operation should respond with an ``OK``. 
+    
+    However, if you receive an error indicating that ``gnupg`` is not
+    installed, you can:
+       
+    #. Install ``gnupg`` and its required libraries using the following command:
+
+       .. code-block:: sh
+
+          sudo apt-get install gnupg
+       
+    #. Once installed, retry importing the key:
+
+       .. code-block:: sh
+
+          wget -qO - https://www.mongodb.org/static/pgp/server-{+pgp-version+}.asc | sudo apt-key add -
 ---
-source:
-  file: steps-install-shell-base.yaml
-  ref: navigate-dlc
-ref: navigate-dlc-linux-deb
+title: Create a list file for MongoDB.
+stepnum: 2
+level: 4
+ref: sources-list
+content: |
+
+   Create the list file
+   ``/etc/apt/sources.list.d/mongodb-org-{+mdb-version+}.list`` for your
+   version of Ubuntu.
+
+   Click on the appropriate tab for your version of Ubuntu.
+   If you are unsure of what Ubuntu version the host is running,
+   open a terminal or shell on the host and execute ``lsb_release -dc``. 
+
+   .. tabs::
+
+      tabs:
+         - id: focal
+           name: Ubuntu 20.04 (Focal)
+           content: |
+
+              The following instruction is for **Ubuntu 20.04 (Focal)**.
+
+              Create the
+              ``/etc/apt/sources.list.d/mongodb-org-{+version+}.list``
+              file for Ubuntu 20.04 (Focal):
+
+              .. code-block:: sh
+
+                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/{+version+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+version+}.list
+                 
+         - id: bionic
+           name: Ubuntu 18.04 (Bionic)
+           content: |
+
+              The following instruction is for **Ubuntu 18.04 (Bionic)**.
+              For Ubuntu 14.04 (Trusty) or Ubuntu 16.04 (Xenial), click
+              on the appropriate tab.
+
+              Create the
+              ``/etc/apt/sources.list.d/mongodb-org-{+mdb-version+}.list``
+              file for Ubuntu 18.04 (Bionic):
+
+              .. code-block:: sh
+
+                 echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/{+mdb-version+} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{+version+}.list
+
+         - id: xenial
+           name: Ubuntu 16.04 (Xenial)
+           content: |
+
+              The following instruction is for **Ubuntu 16.04 (Xenial)**.
+              For Ubuntu 14.04 (Trusty) or Ubuntu 18.04 (Bionic), click
+              on the appropriate tab.
```

Comment:
We currently only publish for `18.04` and `20.04` - for 16.04 or lower we don't publish any `.deb`s to the PPAs right now. Probably best if we remove those tabs then completely and maybe add a hint at the beginning? Or could we have a "16.04 and lower" tab and tell people there that we are not publishing to PPA for those versions? 🤔